### PR TITLE
Parallelize and fix k6 module tests and lint errors

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

File: js/modules/k6/k6.go [hunk 1]
```diff
@@ -43,14 +45,17 @@ var ErrGroupInInitContext = common.NewInitContextError("Using group() in the ini
...
 func New() *K6 {
 	return &K6{}
 }
 
+// Fail is a fancy way of saying `throw "something"`
 func (*K6) Fail(msg string) (goja.Value, error) {
 	return goja.Undefined(), errors.New(msg)
 }
 
+// Sleep waits for the provides seconds before continuing the execution
```
Comment (imiric) [inline js/modules/k6/k6.go:58]:
```suggestion
// Sleep waits the provided seconds before continuing the execution.
```

Sidenote: shouldn't all these comments end in period?

**Review by imiric — DISMISSED** [review]

File: js/modules/k6/k6.go (hunk 1 above)
Comment (MStoykov) [reply js/modules/k6/k6.go:58]: :shrug: 

I literally wanted to remove all linter errors and wrote something just to remove it :P

Comment (MStoykov) [reply js/modules/k6/k6.go:58]: apparently .. .yes https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

**Review by imiric — APPROVED** [review]

File: js/modules/k6/k6.go [hunk 2]
```diff
@@ -18,6 +18,7 @@
  *
  */
 
+// Package k6 implements the module imported as 'k6' from inside k6
```
Comment (codebien) [inline js/modules/k6/k6.go:21]: This period is missing

**Review by codebien — COMMENTED** [review]
LGTM, just a small thing

**Review by na-- — APPROVED** [review]

**Review by na-- — APPROVED** [review]

**Review by imiric — APPROVED** [review]

**Pull request merged**